test-utils/src/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './act';
+export * from './setupRerender';
```

Comment:
Minor nit: Just inline both functions here. No need to make a file per function (that's a popular approach for some projects though, just not for us :) ).